### PR TITLE
Remove unused parts of SDK configuration

### PR DESCRIPTION
**Pull request by annagrin**
Remove all unused parts of SDK configuration and update their callers. 

This change will help us refactor SDK configuration in the future.

Closes: https://github.com/dart-lang/webdev/issues/1910

### CODE REVIEW

File: dwds/test/sdk_asset_generator_test.dart [hunk 1]
```diff
@@ -21,7 +21,6 @@ void main() {
     late final Directory tempDir;
     late String sdkDirectory;
     late final String soundSdkSummaryPath;
-    late final String librariesPath;
```
Comment (elliette) [inline dwds/test/sdk_asset_generator_test.dart:24]: Why are we removing all these references to `libraries` (here and below)?

Comment (annagrin) [reply dwds/test/sdk_asset_generator_test.dart:24]: I removed librariesPath from `SdkConfiguration` as is it not used in dwds (I removed unnecessary passing of it to the expression compiler service and  dart_uri). 

The tests here are only validating the SDK configuration so they cannot use it now either.

File: dwds/test/sdk_configuration_test.dart [hunk 2]
```diff
@@ -139,14 +128,13 @@ void main() {
...
         weakSdkSummaryPath: weakSdkSummaryPath,
-        librariesPath: librariesPath,
         compilerWorkerPath: compilerWorkerPath,
       );
 
       expect(configuration.sdkDirectory, equals(sdkDirectory));
       expect(configuration.soundSdkSummaryPath, equals(soundSdkSummaryPath));
       expect(configuration.weakSdkSummaryPath, equals(weakSdkSummaryPath));
-      expect(configuration.librariesPath, equals(librariesPath));
+
```
Comment (elliette) [inline dwds/test/sdk_configuration_test.dart:137]: nit: remove new line here

**Review by elliette — APPROVED** [review]

File: dwds/test/sdk_configuration_test.dart (hunk 2 above)
Comment (annagrin) [reply dwds/test/sdk_configuration_test.dart]: done

**Pull request merged**